### ISSUE TITLE
GM.info is a function

### ISSUE THREAD

**Issue by iFelix18** [quoid/userscripts#229]
In other userscript managers `GM.info` is an object, see [Greasemonkey](https://wiki.greasespot.net/GM.info), [Violentmonkey](https://violentmonkey.github.io/api/gm/#gm_info), and [Tampermonkey](https://www.tampermonkey.net/documentation.php#GM_info).

A sample script:
```javascript
// ==UserScript==
// @name         GM.info test script
// @description  Test script
// @version      1.0.0
// @match        *://*/*
// ==/UserScript==

console.log(typeof GM.info) // object
console.log(GM.info.script.name) // GM.info test script

```

_System Information:_
> iOS version: 15.3.1
> Userscripts version: 1.1(17)

**Comment by quoid** [comment]
@iFelix18 thanks for bringing this up, I will update next beta build

**Comment by quoid** [comment]
https://github.com/quoid/userscripts/commit/8849baf33246ee771dfbf1283d6f0bf8eb448e68

**Comment by iFelix18** [comment]
Also, `GM.info.script.author`/`GM_info.script.author` is `undefined`.

**Comment by quoid** [comment]
@iFelix18 I didnt see that in the Greasemonkey documentation, but I will add it. 

I updated the documentation for GM.infohttps://github.com/quoid/userscripts/tree/v4.1#api

**Comment by iFelix18** [comment]
> I didnt see that in the Greasemonkey documentation, but I will add it.

@quoid Yep, it is not present in Greasemonkey, but it works on both Violentmonkey and Tampermonkey.

**Comment by quoid** [comment]
fixed in [v4.1.0, v1.1.0](https://github.com/quoid/userscripts/releases/tag/v4.1.0)
